### ISSUE TITLE
Http4s and main dependencies update

### ISSUE THREAD

**Issue by MarmaladeSky** [http4s/rho#715]
Hi there,
Is there any chance to update Rho to fit the latest version of Http4s?
Let me illustrate what I mean with the PR.

**Comment by monzonj** [comment]
I see that your PR is merged, but we haven't had a release of RHO for a long time. Have you made it work with the last version of http4s? Did you package your own version?

**Comment by MarmaladeSky** [comment]
@monzonj 
>Have you made it work with the last version of http4s?

Yes, it was merged and had been working (feel free to check it with cloning the repo and publishing locally) when I tried it last time.

>Did you package your own version?

Not in the public repos. At first, I planned to publish a fork at Maven Central with mirroring versions of http4s, but then there had been a maintainer assigned, so, I had dropped my idea. As I see now, it is still actual.

**Comment by monzonj** [comment]
Thanks @MarmaladeSky . 
I've actually updated to M37 and opened a PR:  #836 

for the time being i'll use a local distribution like you.
